### PR TITLE
Modify transition to partner

### PR DESCRIPTION
**Pull request by jaysonmc**
Adding some links, and changing focus from product management to agile project management. 

Product management will become more relevant but won't replace project management (I propose) 

Unless my definition of product mgmt is wrong, of course

### CODE REVIEW

File: Strategies/TransitionToPartner.md [hunk 1]
```diff
@@ -82,26 +82,26 @@ This means build automation, test automation, continuous integration, release au
 
 &#042; Architecture compliance automation needs to be further evaluated.
 
-## 5. Shift to Product Management
+## 5. Shift to Agile Project Management
 
 IITB will adopt a product management mindset wherein each Solution is viewed as a product and managed throughout its entire lifespan.
 
 Solutions will no longer be delivered as projects that culminate with a release, but rather as products that are continuously improved through smaller, more frequent releases.
```
Comment (remyb2canada) [inline Strategies/TransitionToPartner.md]: We'll need to change that sentence too. I'd say
Solutions will be delivered via agile projects, which make use of products that are continuously improved through smaller, more frequent releases.

File: Strategies/TransitionToPartner.md [hunk 2]
```diff
@@ -82,26 +82,26 @@ This means build automation, test automation, continuous integration, release au
...
    Responsible for orchestrating the various activities associated with ensuring that a product delivered meets users' needs.
    Product managers also evaluates trends and strategic direction to determine features to build.
-2. **A new financial model**.
-   Product Management requires stable and predictable funding, without foresight as to what features are to be implemented in the next year.
-3. **A dedicated team**.
-   Product Management requires dedicated teams that understands and knows the product so that it can continuously improve, manage technical debt, and add new features to the product.
-   Teams cannot be moved around and assigned to different projects as the winds of change occurs, they must remain attached to products.
-4. **Address technical debt as continuous improvement**
-   Dedicated teams will address technical debt not as change requests or small projects, but rather as continuous improvement measures built within the team itself in collaboration with the product owner and manager.
+2. **Agile financial model**
```
[the remark this answers is not in the document]
Comment (jaysonmc) [reply Strategies/TransitionToPartner.md]: Good with #2 

#1 ... I'm not sure if we should comment on the finances. We just say, from an IT perspective, we should do this. If the finance needs to be updated or not isn't really our business or concern. Thoughts?

**Comment by remyb2canada** [comment]
@jaysonmc , please review changes and approve (split 1 file into 7 for linking to StrategySummary roadmap)
:smiley:

**Comment by gcharest** [comment]
@remyb2canada Not sure why we're breaking it in sub documents but OK. Could have used the in page links instead.

**Comment by remyb2canada** [comment]
@gcharest doing some micro-md so that future changes are easier :laughing:

**Comment by gcharest** [comment]
OK but think about how this will render as a website as well ;)

For example, could have used this: 
image

It adds the location URL in the page: 
image

**Comment by gabrielcossette** [comment]
I'm not sure either why we should split the Strategy in different files. Could you share more details on the advantages @remyb2canada ?

**Comment by remyb2canada** [comment]
@gabrielcossette @gcharest cool, reverted to last commit :)
File is back to one file only

**Pull request merged**